### ISSUE TITLE
Improve Credentials documentation to cover container credentials

### ISSUE THREAD

**Issue by pditommaso** [seqeralabs/nf-tower-docs#259]
The [credentials documentation](https://help.tower.nf/22.2/credentials/overview/) should be extended to include the support for container registries. 

<img width="1391" alt="Screenshot 2022-10-06 at 09 49 26" src="https://user-images.githubusercontent.com/816968/194248658-53b3d8dd-5faa-4ea7-88a4-8e67c4773fb4.png">

Worth mentioning, that until recently it was possible to use these credentials only with Azure compute environment. In the latest release this capability has been extended to any compute environment supporting the Wave feature. 

The documentation should ideally shows how to configure container credentials for those registries:
* docker.io
* quay.io 
* AWS ECR
* Google Artifactory 
* Azure containers

**Comment by llewellyn-sl** [comment]
@pditommaso is this the complete list of container registry credentials we will support as of v22.4?

docker.io
quay.io
AWS ECR
Google Artifactory
Azure containers

**Comment by pditommaso** [comment]
Yes. Here the critical piece of information is how to configure each of them. For example Docker uses [personal access token](https://docs.docker.com/docker-hub/access-tokens/). Quay.io is somehow the same. With AWS ECR, users must provide AWS keys and grant some magic IAM permissions. etc

Think, we'll need a separate section for each of them
